### PR TITLE
Feature/integrate ccd setup

### PR DESCRIPTION
**Pull request by masahmad**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/app/case/case-feature/case-feature.config.ts [hunk 1]
```diff
@@ -0,0 +1,107 @@
+import { Injectable } from '@angular/core'
+import { Http } from '@angular/http'
+import { AbstractAppConfig, CaseEditorConfig, CaseEventData } from '@hmcts/ccd-case-ui-toolkit'
+// import { environment as mainConfig } from '../environments/environment'
+//    'case_data_url': 'http://localhost:3001/api/data',
+@Injectable()
+export class AppConfig extends AbstractAppConfig {
+    protected config: CaseEditorConfig = {
+        'api_url': '/aggregated',
```
Comment (frontr-uk) [inline src/app/case/case-feature/case-feature.config.ts]: suggest moving this to environment file.

File: src/app/case/case-feature/container/create-case/create-case.component.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+  caseSelected: string
+  constructor(
+    //private logger: LoggerService
+  ) {
+  }
+
+  ngOnInit() {
+
+    //this.logger.info('CreateCaseComponent: initialized');
```
Comment (frontr-uk) [inline src/app/case/case-feature/container/create-case/create-case.component.ts]: we can remove this if it is commented out.

File: src/app/case/case-feature/case-feature.config.ts (hunk 1 above)
Comment (masahmad) [reply src/app/case/case-feature/case-feature.config.ts]: yes good idea - moved it

File: src/app/case/case-feature/container/create-case/create-case.component.ts (hunk 2 above)
Comment (masahmad) [reply src/app/case/case-feature/container/create-case/create-case.component.ts]: done

**Pull request merged**